content/swiftui/concepts/views/terms/lazyHStack/lazyHStack.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'LazyHStack'
+Description: 'The LazyHStack view is a layout container where elements are arranged in a horizontal line (left-to-right) and are created when needed.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Stacks'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`LazyHStack`** view is a layout container where items are arranged in a horizontal line (left-to-right), like in a standard [`HStack`](https://www.codecademy.com/resources/docs/swiftui/views/hstack) view, but will only create them when needed. This is useful for creating long lists and can allow an app to run smoother and faster.
```

Comment:
```suggestion
The **`LazyHStack`** view is a layout container where items are arranged in a horizontal line (left-to-right), like in a standard [`HStack`](https://www.codecademy.com/resources/docs/swiftui/views/hstack) view, but it will only create them when needed. This is useful for creating long lists and can allow an app to run smoother and faster.
```